addons/mail/models/mail_activity.py

```diff
@@ -597,17 +641,33 @@ def get_activity_data(self, res_model, domain, limit=None, offset=0):
         if domain or limit or offset:
             res = self.env[res_model].search(domain or [], limit=limit, offset=offset)
             activity_domain.append(('res_id', 'in', res.ids))
+        if 'activities_my' in activity_filters:
+            activity_domain.append(('user_id', '=', self.env.uid))
+        if 'activity_user_id' in activity_search_fields:
+            activity_domain.append(('user_id.name', 'ilike', activity_search_fields['activity_user_id']))
+        if 'activity_create_uid' in activity_search_fields:
```

Comment:
Is it really relevant ? I mean, generally you want to find assigned activities... what is the use case of finding records having activities created by someone ? Would avoid adding a new field + computation that could be really slow, an ilike on user name ...